src/main/resources/application.yml

```diff
@@ -0,0 +1,7 @@
+server:
+  port: 8080
+  ssl:
+    key-store: classpath:keystore.jks
```

Comment:
Another keystore.jks? Keys shipped in production code?

Reply:
One for integration-tests and one for development.

Reply:
Production run is supposed to get the keys from somewhere out of the war file.
Any reason we cannot get it in integration test run  from a location which happens to be the resource directory of the test code?